--- xyzservices-2021.11.0.orig/setup.py	2021-11-05 20:48:54.000000000 -0300
+++ xyzservices-2021.11.0/setup.py	2021-12-02 15:08:54.940323291 -0300
@@ -14,7 +14,7 @@
     author="Dani Arribas-Bel, Martin Fleischmann",
     author_email="daniel.arribas.bel@gmail.com, martin@martinfleischmann.net",
     license="3-Clause BSD",
-    packages=setuptools.find_packages(),
+    packages=setuptools.find_packages(exclude=["tests"]),
     python_requires=">=3.7",
     include_package_data=True,
     package_data={
